meiki_server/auth/auth_test.go

```diff
@@ -32,7 +32,7 @@ func (s *AuthTestSuite) clean() {
 func (s *AuthTestSuite) SetupTest() {
 	log.Initialize()
 
-	s.ctx, s.cancel = context.WithTimeout(context.Background(), 500*time.Millisecond) // 500ms might not be enough on some systems?
+	s.ctx, s.cancel = context.WithTimeout(context.Background(), 500000000000*time.Millisecond) // 500ms might not be enough on some systems?
```

Comment:
does this prevent it from timing out in situations where the test services are not initialized?

Reply:
Yeah, will change back

Reply:
Addressed in next PR